frontend/src/scenes/billing/BillingSection.scss

```diff
@@ -0,0 +1,31 @@
+.BillingSection {
```

Comment:
We're trying to move away from scss so it's best to use inline tailwind styling now. LLMs are good at converting this if you need help.

Reply:
Removed, thanks!